terrautils/metadata/lemnatec.py

```diff
@@ -1,24 +1,32 @@
-'''
-Create a cleaned/curated metadata object based on the LemnaTec fixed metadata
-Some metadata used for search
-Some metadata used for extractors
+"""
+Given a JSON object containing the original LemnaTec metadata, convert
+field names and values to standardized formats required for TERRA-REF.
 
-TODO: 
-  Add checks for expected fields
-  Date/time standardization
-  Add units to variables?
-  Logging
+The "cleaned" metadata will be posted to the Clowder metadata endpoint and
+used downstream by extractors.
 
-'''
+TODO:
+    SENSOR_IRRIGATION                                                                                   
+    SENSOR_LIGHTNING
+    SENSOR_ENVIRONMENTAL_LOGGER
+    SENSOR_WEATHER
+"""
 
+import argparse
 import json
+import logging
 import os
+import pytz, datetime
 import requests
-from terrautils import sensors
+import sys
+from terrautils.sensors import Sensors
+
+STATION_NAME = "ua-mac"
+TERRAREF_BASE='/projects/arpae/terraref/sites'
```

Comment:
Terrautils is designed to get fields like TERRAREF_BASE from the environment with a possible override from the command line. We want this to be consistent across all the extractors and not have different extractors using different names or approaches.

This makes it easy to manage from a desktop machine, a virtual machine, docker containers or ROGER's interactive node; all of which may have a different view of the filesystem and allows us to easily move between filesystems.